src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -7893,15 +7893,27 @@ public void SuppressionOnNullableReferenceType_AppliedOnField2()
 public class C
 {
     public string? field;
-    void M(C? c)
+    void M1(C? c)
     {
         c?.field!.ToString();
-        c.ToString(); // We learned from suppressed dereference that `c` isn't null
+        c.ToString(); // 1
```

Comment:
@jcouv I wasn't sure whether we actually want 'c' to become not-null due to the suppression on `field!` any more.